scripts/actions/mobskills/frozen_mist.lua

```diff
@@ -0,0 +1,36 @@
+-----------------------------------
+--  Frozen Mist
+--  Description: Deals ice damage to enemies around the caster. Additional effect: Terror
+--  Type: Magical (Ice)
+-----------------------------------
+local mobskillObject = {}
+
+mobskillObject.onMobSkillCheck = function(target, mob, skill)
+    return 0
+end
+
+mobskillObject.onMobWeaponSkill = function(target, mob, skill)
+    local duration = 30
+    local resist = applyResistanceAddEffect(mob, target, xi.element.ICE, 0)
```

Comment:
This always seems to return 1, resulting in a max duration every mobskill. Am I doing something wrong here?